tosca/constants.go

```diff
@@ -25,3 +25,9 @@ const RunnableRunOperationName = RunnableInterfaceName + ".run"
 
 // RunnableCancelOperationName is the fully qualified name of the Cancel operation
 const RunnableCancelOperationName = RunnableInterfaceName + ".cancel"
+
+// StandardInterfaceName is the fully qualified name of the Standard interface
+const StandardInterfaceName = "tosca.interfaces.node.lifecycle.standard"
+
+// StandardInterfaceShortName is the short name of the Standard interface
+const StandardInterfaceShortName = "standard"
```

Comment:
No need for a short version of runnable interface?
Is this not impacted by the short/long naming issue?

Reply:
I don't find any usage of "short" version of runnable interface. So, right now, there is no need IMO of a short version of runnable interface.

Reply:
OK thanks for the clarification :+1:

Reply:
Well sorry but what about relationship operations `tosca.interfaces.relationships.configure`?
I think they are handled in the same way by Alien4Cloud.

Reply:
Yes, I add this case